style.css

```diff
@@ -0,0 +1,117 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+.hamburger-menu {
+  width: 100%;
+  height: 74px;
+  background: url('assets/Shape.png') no-repeat, url('assets/Shape-2.png') no-repeat 10% 12%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  padding: 19px 14px 0 0;
+}
+
+.bars {
+  width: 24px;
+  height: 18px;
+  margin-right: 23px;
+}
+
+.bars:hover {
+  background-color: blue;
+}
+
+.bars1 {
+  position: absolute;
+  top: 19px;
+  left: 330px;
+  background-color: #000;
+  width: 20px;
+  height: 2px;
+  border: 0 solid black;
+}
+
+.bars2 {
+  position: absolute;
+  top: 25px;
+  left: 330px;
+  background-color: #000;
+  width: 20px;
+  height: 2px;
+  border: 0 solid black;
+}
+
+.bars3 {
+  position: absolute;
+  top: 31px;
+  left: 330px;
+  background-color: #000;
+  width: 20px;
+  height: 2px;
+  border: 0 solid black;
+}
+
+.hello {
+  width: 330px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.about-me {
+  padding: 30px 0 0 16px;
+}
+
+.about-me h1 {
+  width: 320px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+  margin-top: 0;
+  margin-bottom: 17px;
+}
+
+.about-me p {
+  width: 320px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+  margin: 0;
+}
+
+#collaboration-btn {
+  width: 320px;
+  background-color: #1a2236;
+  color: #fff;
+  height: 50px;
+  margin-top: 22px;
+}
+
+#collaboration-btn:active {
+  border-color: blue;
+}
+
+#collaboration-btn:hover {
+  border-color: blue;
+  border-inline-color: red;
```

Comment:
- [ ] kindly consider giving this button the active and hover effect which has the same colors as the one given in the Figma design to make your project look exactly as the given one.  choose one to go with  from the below screen shoot 👇 

<img width="615" alt="Screen Shot 2023-04-27 at 19 47 05" src="https://user-images.githubusercontent.com/60146030/234949777-68931c06-2c3e-4bb7-8a70-76fc5622e49c.png">